package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gh-user",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
Generally don't bump the version in PRs. Owners have their own release process ;)